gammapy/irf/effective_area_table.py

```diff
@@ -1,14 +1,19 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
 import numpy as np
+import matplotlib.pyplot as plt
 from astropy import log
 from astropy.io import fits
 from astropy.units import Quantity
+from astropy.coordinates import Angle, SkyCoord
 from astropy.table import Table
 from ..extern.validator import validate_physical_type
 from ..utils.array import array_stats_str
+from scipy.interpolate import LinearNDInterpolator, RectBivariateSpline
```

Comment:
At least at the moment `scipy` is an optional dependency and you have to move this import into the method where you use it.